tools/building.py

```diff
@@ -1357,20 +1356,23 @@ def map_to_js_libs(library_name):
     'pthread': [],
     'X11': ['library_xlib.js'],
     'SDL': ['library_sdl.js'],
-    'stdc++': [],
     'uuid': ['library_uuid.js'],
     'websocket': ['library_websocket.js']
   }
+  # And some are hybrid and require JS and native libraries to be included
+  native_library_map = {
+    'GL': 'libgl',
+  }
 
   if library_name in library_map:
     libs = library_map[library_name]
     logger.debug('Mapping library `%s` to JS libraries: %s' % (library_name, libs))
-    return libs
+    return (libs, native_library_map.get(library_name))
 
   if library_name.endswith('.js') and os.path.isfile(path_from_root('src', 'library_' + library_name)):
-    return ['library_' + library_name]
+    return (['library_' + library_name], None)
 
-  return None
+  return (None, None)
```

Comment:
perhaps add a comment at the top of the function explaining what the return value is now, as this PR makes it a tuple?

Reply:
Done.